src/ServerlessAutoSwagger.ts

```diff
@@ -104,6 +104,7 @@ export default class ServerlessAutoSwagger {
     const autoswagger = this.serverless.service.custom?.autoswagger ?? {};
 
     if (autoswagger.basePath) this.swagger.basePath = autoswagger.basePath;
+    if (autoswagger.host) this.swagger.host = autoswagger.host;
```

Comment:
Could you add this test case to the unit test `should use overrides if specified`? Should be as simple as going into `tests/ServerlessAutoSwagger.test.ts` and adding `host: 'some-host'` below line [821](https://github.com/SamWSoftware/serverless-auto-swagger/blob/3ccd0394bca8f7e89dbcdc86db1241c7c3f529e3/tests/ServerlessAutoSwagger.test.ts#L821), then the same below line [839](https://github.com/SamWSoftware/serverless-auto-swagger/blob/3ccd0394bca8f7e89dbcdc86db1241c7c3f529e3/tests/ServerlessAutoSwagger.test.ts#L839)